cmd/access/node_builder/unstaked_access_node_builder.go

```diff
@@ -234,7 +232,7 @@ func (anb *UnstakedAccessNodeBuilder) enqueueMiddleware(ctx context.Context) {
 			// for now we use the empty metrics NoopCollector till we have defined the new unstaked network metrics
 			unstakedNetworkMetrics := metrics.NewNoopCollector()
 
-			libP2PFactory, err := anb.initLibP2PFactory(ctx, unstakedNodeID, unstakedNetworkKey)
```

Comment:
remove `ctx` from `enqueueMiddleware` as well?

Reply:
Great catch, looks like we can remote context entirely :)